Dockerfile

```diff
@@ -2,12 +2,14 @@ ARG PYTHON_VERSION=3.7
 
 FROM python:${PYTHON_VERSION}
 
+ENV PIP_OPTIONS "--no-cache-dir --progress-bar off"
+
 RUN apt-get update \
     && apt-get -y install openmpi-bin libopenmpi-dev \
-    && apt-get -y install swig  \
     && rm -rf /var/lib/apt/lists/* \
     && pip install --no-cache-dir -U pip \
-    && pip install --no-cache-dir --progress-bar off -U setuptools
+    && pip install ${PIP_OPTIONS} -U setuptools \
+    && pip install ${PIP_OPTIONS} Cython  # for automl/ConfigSpace
```

Comment:
Is `ConfigSpace` still required? I know that it was used with `fanova` before.

Reply:
Let me see whether we can remove ConfigSpace... I could not catch up recent active development enough :crying_cat_face:

Reply:
No worries, of course (unless we can improve the situation somehow). To elaborate a bit, this is the recent change I was referring to https://github.com/optuna/optuna/pull/1106/files#diff-2eeaed663bd0d25b7e608891384b7298.